cmake/templates/HPXMacros.cmake.in

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2014 Thomas Heller
-# Copyright (c) 2015 Andreas Sch�fer
+# Copyright (c) 2015 Andreas Sch�fer
```

Comment:
Please change this to use `ae` (ASCII characters only)

Reply:
Done